crates/moonutil/src/scan.rs

```diff
@@ -274,6 +274,18 @@ fn scan_one_package(
         wbtest_mbt_files.sort();
         test_mbt_files.sort();
     }
+    // if warn_list & alert_list not set in current moon.pkg.json, use the one in moon.mod.json
+    let warn_list = if pkg.warn_list.is_none() {
```

Comment:
Should pkg.warn_list be merged with mod's warn_list or just replaced?

Reply:
let's append